add-ons/plugin.video.streamz
# Improve tests

Tests:
* Replace `coverage` with `pytest-cov`
* Remove `tox`
* Add tests for Library
* Run service with pylint instead of starting it directly

Github Actions:
* Don't run actions twice (push & pull_request), see https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662
* Use `xbmc/action-kodi-addon-checker@v1.2` action instead of installing ourself
* Use output of `xbmc/action-kodi-addon-submitter` for the Github release
